code/ship/ship.cpp

```diff
@@ -6804,8 +6821,8 @@ void ship_render_show_ship_cockpit(object *objp)
 
 	render_info.set_object_number(OBJ_INDEX(objp));
 	render_info.set_detail_level_lock(0);
-
-	model_render_immediate(&render_info, Ship_info[Ships[objp->instance].ship_info_index].model_num, &objp->orient, &vmd_zero_vector); // Render ship model with fixed detail level 0 so its not switching LOD when moving away from origin
+	// Render ship model with fixed detail level 0 so its not switching LOD when moving away from origin
+	model_render_immediate(&render_info, sip->model_num, &objp->orient, &vmd_zero_vector, 3, true, pos_for_detail_box_check, true);
```

Comment:
It would be better to use MODEL_RENDER_ALL here instead of the magic number '3'

Reply:
Good point.